tests/ggsmark.spec.js

```diff
@@ -191,47 +191,83 @@ describe('render youtube blocks', () => {
     expect(result).toMatchSnapshot()
   })
 })
+describe('render left allignment text', () => {
+  test('single line', () => {
+    // Arrange
+    let string = dedent`
+    <-LeftText<-
+    `
+
+    // Act
+    let result = ggsmark(string)
 
-describe('do not render custom html', () => {
-  test('span', () => {
+    // Assert
+    expect(result).toMatchSnapshot()
+  })
+  test('multi line', () => {
     // Arrange
     let string = dedent`
-    <span style="color: red">Test</span>
+    <-This is a multiple line texts\n 2nd line here<-
     `
+
     // Act
     let result = ggsmark(string)
 
     // Assert
-    expect(result).toBe(dedent`
-    <p>Test</p>
-    `)
+    expect(result).toMatchSnapshot()
   })
+})
 
-  test('heading', () => {
+describe('render center allignment text', () => {
+  test('single line', () => {
     // Arrange
     let string = dedent`
-    <h3 style="color: red">Test</h3>
+    ->CenterText<-
     `
+
     // Act
     let result = ggsmark(string)
 
     // Assert
-    expect(result).toBe(dedent`
-    <p>Test</p>
-    `)
+    expect(result).toMatchSnapshot()
+  })
+  test('multi line', () => {
+    // Arrange
+    let string = dedent`
+    ->This is a multiple line texts\n 2nd line here<-
+    `
+
+    // Act
+    let result = ggsmark(string)
+
+    // Assert
+    expect(result).toMatchSnapshot()
   })
+})
 
-  test('div', () => {
+describe('render right allignment text', () => {
+  test('single line', () => {
     // Arrange
     let string = dedent`
-    <div style="color: red">Test</div>
+    ->RightText->
     `
+
     // Act
     let result = ggsmark(string)
 
     // Assert
-    expect(result).toBe(dedent`
-    <p>Test</p>
-    `)
+    expect(result).toMatchSnapshot()
+  })
+  test('multi line', () => {
+    // Arrange
+    let string = dedent`
+    ->This is a multiple line texts\n 2nd line here->
+    `
+
+    // Act
+    let result = ggsmark(string)
+
+    // Assert
+    expect(result).toMatchSnapshot()
   })
 })
```

Comment:
```suggestion
})
describe('do not render custom html', () => {
  test('span', () => {
    // Arrange
    let string = dedent`
    <span style="color: red">Test</span>
    `
    // Act
    let result = ggsmark(string)

    // Assert
    expect(result).toBe(dedent`
    <p>Test</p>
    `)
  })

  test('heading', () => {
    // Arrange
    let string = dedent`
    <h3 style="color: red">Test</h3>
    `
    // Act
    let result = ggsmark(string)

    // Assert
    expect(result).toBe(dedent`
    <p>Test</p>
    `)
  })

  test('div', () => {
    // Arrange
    let string = dedent`
    <div style="color: red">Test</div>
    `
    // Act
    let result = ggsmark(string)

    // Assert
    expect(result).toBe(dedent`
    <p>Test</p>
    `)
  })
})
```
Add this test back, the merge is getting confused with your code